content/news/buzz/august-2024-status-update/contents.lr

```diff
@@ -0,0 +1,57 @@
+title: August 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+August has seen some good progress on BeeWare's support for Python 3.13, plus a number of long-awaited features landing in a stable release of Toga.
+
+What we've done
+----------------
+
+* We've completed the process of backporting the CPython changes for iOS compatibility to Python 3.9 through 3.12. We've published support packages and stub binaries that incorporate these changes, and made the `changes to Briefcase needed to support these new versions <https://github.com/beeware/briefcase/pull/1934>`__.
+
+* We've begun work on updating Mobile Forge and related binary tooling to support the new official binary tag and framework formate required by the PEP-compliant patch for iOS support. So far, we're able to compile simple binary packages like ``lru-dict``, and non-Python dependencies like ``libjpeg``; we're working on restoring binary support for more complex binary dependencies.
+
+* We've landed the `Android test runner script <https://github.com/python/cpython/pull/121595>`__, fixed a number of issues that have arisen in CPython's Android support since the original patches landed, and `deployed an Android CPython buildbot <https://buildbot.python.org/#/builders/1594>`__. This buildbot has revealed some edge cases (primarily in socket and signal handling) that only appear to manifest under the unusual load conditions generated by a buildbot, so we've temporarily disabled the buildbot while we investigate the cause of these problems.
```

Comment:
```suggestion
* We've landed the `Android test runner script <https://github.com/python/cpython/pull/121595>`__, fixed a number of issues that have arisen in CPython's Android support since the original patches landed, and `deployed an Android CPython buildbot <https://buildbot.python.org/#/builders/1594>`__. This buildbot has revealed some edge cases (primarily in socket and signal handling) that only appear to manifest under the unusual load conditions generated by a buildbot; so, we've temporarily disabled the buildbot while we investigate the cause of these problems.
```